CMakeLists.txt

```diff
@@ -9,29 +9,25 @@ project(editorconfig-core-qt
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core REQUIRED)
-
-enable_testing()
-set(EDITORCONFIG_CMD ${CMAKE_BINARY_DIR}/src/editorconfig-app/editorconfig-app)
-add_subdirectory(tests)
-
-if (BUILD_STATIC)
-    set(CMAKE_STATIC_LIBRARY_SUFFIX "_static${CMAKE_STATIC_LIBRARY_SUFFIX}")
-    add_library(editorconfig-core-qt STATIC
-        EditorConfig.cpp
-        EditorConfig.h
-    )
-else()
-    add_library(editorconfig-core-qt SHARED
-        EditorConfig.cpp
-        EditorConfig.h
-    )
-endif()
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core)
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core)
+
+add_library(editorconfig-core-qt
+    EditorConfig.cpp
+    EditorConfig.h
+)
 
-target_link_libraries(editorconfig-core-qt PRIVATE Qt::Core)
+target_link_libraries(editorconfig-core-qt PRIVATE Qt${QT_VERSION_MAJOR}::Core)
```

Comment:
On Qt 5.12 and Qt6 - this is not needed.

Reply:
I copied some code to support older Qt versions but since Qt5.15 is the oldest supported, then yes I agree this change isn't needed.